helm/extensions.bzl

```
@@ -93,8 +102,8 @@ def _register_go_yaml():
         build_file = Label("//3rdparty/yaml:BUILD.yaml.bazel"),
     )
 
-options = tag_class(
-    doc = "Options used to configure helm toolchains.",
+toolchain = tag_class(
```

I think this should be private so it's not loadable externally.
```suggestion
_toolchain = tag_class(
```